tests/components/startca/test_sensor.py

```diff
@@ -150,47 +151,47 @@
     assert state.state == "0"
 
     state = hass.states.get("sensor.start_ca_usage")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "0.0"
 
     state = hass.states.get("sensor.start_ca_data_limit")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "inf"
 
     state = hass.states.get("sensor.start_ca_used_download")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "0.0"
 
     state = hass.states.get("sensor.start_ca_used_upload")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "0.0"
 
     state = hass.states.get("sensor.start_ca_used_total")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "0.0"
 
     state = hass.states.get("sensor.start_ca_grace_download")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "304.95"
 
     state = hass.states.get("sensor.start_ca_grace_upload")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "6.48"
 
     state = hass.states.get("sensor.start_ca_grace_total")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "311.43"
 
     state = hass.states.get("sensor.start_ca_total_download")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "304.95"
 
     state = hass.states.get("sensor.start_ca_total_upload")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "6.48"
 
     state = hass.states.get("sensor.start_ca_remaining")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5001170)